src/app/portlets/content-types/fields/service/field.service.ts

```diff
@@ -29,40 +29,35 @@ export class FieldService {
      * @memberof FieldService
      */
     saveFields(contentTypeId: string, fields: Field[]): Observable<any> {
-        const observables: Observable<any>[] = fields.map((field, index) => {
-            const fieldToSend = Object.assign({}, field, {
-                contentTypeId: contentTypeId,
-                sortOrder: index + 1
-            });
+        fields.forEach((field, index) => {
+            field.contentTypeId = contentTypeId;
```

Comment:
Am I getting this wrong or we are assign it the same ```contentTypeId``` to all the fields?

Reply:
Yes, it is that, the backend need it